### ISSUE TITLE
When running --init, if --box-id is not specified, increment the last used id instead of defaulting to 0 and throwing an error

### ISSUE THREAD

**Issue by whiteShtef** [ioi/isolate#60]

**Comment by gollux** [comment]
I do not think this is a good idea. It would completely break if Isolate is used by several processes in parallel.

**Comment by whiteShtef** [comment]
@gollux why do you think that? Correct me if I'm wrong, but all you need to do in order to make it work in parallel is to wrap a mutex around the "latest id" var.

**Comment by gollux** [comment]
Well, it is certainly not so simple: the counter would need to be shared 
between different invocations of isolate, so we would need a shared file
with some file-level locking.

However, I objected to something else: When "isolate --init" allocates some unused box ID, how do you know which ID to pass to "isolate --run" and "isolate --clean" later?
